db/migrate/20150312144232_migrate_websockets_setting.rb

```diff
@@ -0,0 +1,19 @@
+class MigrateWebsocketsSetting < ActiveRecord::Migration
+  def up
+    return unless encrypt = Setting.find_by_name("websockets_encrypt")
+    encrypt.settings_type = "boolean"
+    if encrypt.value == "auto"
+      encrypt.value = (Setting[:websockets_ssl_key].nil? ||
```

Comment:
> Nitpick, the old test was for `.blank?` rather than `.nil?`, since a user might set it to an empty value.

Empty sets nil, but sure, that's better.

-- 
Later,
 Lukas #lzap Zapletal